dev/com.ibm.ws.microprofile.metrics.1.1_fat_tck/publish/servers/MetricsTCKServer/server.xml

```diff
@@ -0,0 +1,31 @@
+<server>
+    <featureManager>
+    	<feature>mpMetrics-1.1</feature>
+    	<feature>mpConfig-1.2</feature>
```

Comment:
Do we still need mpConfig-1.2 as we already have com.ibm.websphere.appserver.mpConfig-1.2 in the feature file.